packages/server/api/src/app/database/seeds/openops-delete-old-opportunities-table.ts

```diff
@@ -38,31 +46,28 @@ export const deleteOldOpportunitiesTable = async (): Promise<void> => {
     return;
   }
 
+  if (await newTableWasAlreadyCreated()) {
+    await setOpportunitiesTableDeleted();
+    return;
+  }
+
   try {
     const { token } = await authenticateDefaultUserInOpenOpsTables();
 
-    const opportunitiesTableIdOld = await getTableIdByTableName(
-      SEED_OPENOPS_TABLE_NAME,
-    );
-
-    await makeOpenOpsTablesDelete<unknown>(
-      `api/database/tables/${opportunitiesTableIdOld}/`,
-      createAxiosHeaders(token),
-    );
-    await setOpportunitiesTableDeleted();
-  } catch (error: unknown) {
-    if (
-      error instanceof Error &&
-      error.message === `Table '${SEED_OPENOPS_TABLE_NAME}' not found`
-    ) {
+    const table = await getTableByName(SEED_OPENOPS_TABLE_NAME);
+    if (!table) {
       logger.info('Skip: OpenOps deletion of old opportunities table', {
         name: 'deleteOldOpportunitiesTable',
       });
     } else {
-      logger.error(
-        'An error occurred deleting old opportunities table.',
-        error,
+      await makeOpenOpsTablesDelete<unknown>(
+        `api/database/tables/${table.id}/`,
+        createAxiosHeaders(token),
       );
     }
+
+    await setOpportunitiesTableDeleted();
```

Comment:
setOpportunitiesTableDeleted = setOldOpportunitiesTableDeleted

Just to clarify